interface/billing/billing_process.php

```diff
@@ -46,6 +46,12 @@
   $BILLING_EXPORT = true;
 }
 
+// If UB-04 or Both selected in claim type include these files
+if ($GLOBALS['claim_type'] =='1' || $GLOBALS['claim_type'] =='2') {
+  include_once("$srcdir/gen_837i.inc.php");
```

Comment:
It would be nice if you could do a cms1450 like you did the 1500 and then we can work on the billing changes